src/core/storage/compression/Benchmarks.cpp

```diff
@@ -0,0 +1,64 @@
+#include <core/storage/compression/Compressor.h>
+#include <core/storage/compression/Decompressor.h>
+#include <benchmark/cppbenchmark.h>
+
+namespace fs = std::filesystem;
+
+//	67108864 for 1GB txt file
+constexpr int file_size = (32);
```

Comment:
Why the brackets :smile: ?

Reply:
I don't understand what you mean

Reply:
Why `= (32)` and not `= 32` or `(32)` :smile: It is not a big deal.